translations/ui-users/en_US.json

```diff
@@ -413,14 +413,17 @@
     "accounts.closed": "Closed",
     "accounts.all": "All",
     "accounts.numOpenAccounts": "{count, number} {count, plural, one {open fee/fine} other {open fees/fines}}",
+    "accounts.totalOpenAccounts": "(Total: {amount})",
```

Comment:
New values need to be added to `en.json` only. The transfer from Lokalise to GitHub is one-way and changes to any other locale file will be overwritten.